sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -729,6 +732,13 @@ public void validate() {
     private Instant curTimestamp;
     private Iterator<PartitionState> curBatch = Collections.emptyIterator();
 
+    private final Counter messagesConsumed = Metrics.counter("io", "messagesConsumed");
+    private final Counter bytesConsumed = Metrics.counter("io", "bytesConsumed");
```

Comment:
From my perspective:

1. Having a standard namespace and standard metrics within that namespace makes sense. Ideally, a specific namespace * name pair will always have the same meaning across sources that use it.
1. In the fullness of time, it would be great to turn that into a cleaner "report IO metrics" API that reports a more structured set of information, but we don't have that API, so using metrics like this makes sense.